qemu/tests/qemu_guest_agent.py

```diff
@@ -1299,21 +1299,42 @@ def _guest_cmd_run(guest_cmd, cmd_args=None, env_qga=None,
             if not result["exited"]:
                 test.error("Guest cmd is still running, pls login guest to"
                            " handle it or extend your timeout.")
-            # check the exitcode and output/error data result
-            if result["exitcode"] == 0:
+            # check the exitcode and output/error data if capture_output
+            #  is true
+            if capture_output is not True:
+                return result
```

Comment:
@xiagao  capture_output is not True,  only return result.   If have some error message in result.  maybe you cann't find it.  And case test result is Pass.   We don't need to deal with 'result'?  only return it.

Reply:
@xiangchunfu 
1. the result is output.
2. this code is a sub function, only need return the result.